zebra-consensus/src/checkpoint.rs

```diff
@@ -607,8 +608,15 @@ where
             crate::block::check::equihash_solution_is_valid(&block.header)?;
         }
 
+        let expected_block_subsidy = block_subsidy(height, &self.network)?;
+
+        // TODO: Add link to lockbox stream ZIP
+        let expected_deferred_amount =
+            funding_stream_values(height, &self.network, expected_block_subsidy)?
+                .remove(&FundingStreamReceiver::Deferred);
```

Comment:
This could cause a [panic](https://github.com/ZcashFoundation/zebra/actions/runs/10181559322/job/28162027058?pr=8729) because `block_subsidy()` expects to be called with heights after the slow start interval.

```suggestion
        // TODO: Add link to lockbox stream ZIP
        let expected_deferred_amount = if height > self.network.slow_start_shift() {
            funding_stream_values(height, &self.network, block_subsidy(height, &self.network)?)?
                .remove(&FundingStreamReceiver::Deferred)
        } else {
            None
        };
```